rclpy/src/rclpy/_rclpy_action.cpp

```diff
@@ -79,10 +81,8 @@ rclpy_action_destroy_entity(py::capsule pyentity, py::capsule pynode)
   if (ret != RCL_RET_OK) {
     std::string error_text = "Failed to fini '";
     error_text += pyentity.name();
-    error_text += "': ";
-    error_text += rcl_get_error_string().str;
```

Comment:
I think it is okay to replace `std::runtime_error` into `rclpy::RCLError`. but this `rcl_get_error_string` is supposed to get and include the error messages in `rcl` and `rmw` to tell the application which layer and why the API is failed. i am not sure how come deleting this messages is gonna help us more.

Reply:
The point of `rclpy::RCLError` is to encapsulate the logic for retrieving and clearing the RCL error, and reporting it as part of the exception. This PR should result in no changes in behavior, beyond the exception type changing. The messages should stay the same.

https://github.com/ros2/rclpy/blob/3e030c09a3fbb1a5c4ac1f7d827c1deffff5d242/rclpy/src/rclpy_common/src/exceptions.cpp#L27

Reply:
Ah I see, my oversight...thanks for the study 👍